benchmarks/bench_multi_gpu.py

```diff
@@ -152,8 +152,7 @@ def run_n_iterations(n, fn):
     cleanup()
 
 
-def run():
-    compile = True
+def run(compile: bool):
```

Comment:
How does `compile` get passed into this now?

Reply:
it was hardcoded, just making it configurable from command line